src/main/java/com/exactpro/th2/common/schema/factory/AbstractCommonFactory.java

```diff
@@ -246,6 +254,51 @@ public GrpcRouter getGrpcRouter() {
         });
     }
 
+    /**
+     * Registers message router for custom type that is passed via {@code messageClass} parameter. <br>
+     *
+     * Creates the router at the first call. The next calls with the same {@code messageClass} will return the same router as the first call did.
+     * The router will be automatically closed when {@link #close()} is invoked
+     * @param messageClass custom message class
+     * @param messageConverter converter to serialize message to bytes and vice versa
```

Comment:
@param messageConverter converter **used** to serialize message to bytes and vice versa